outline.md

```diff
@@ -0,0 +1,62 @@
+* Introduction
+  - A Brief History of Open
+  - Types of Digital Public Goods
+    + Software
+    + Standards
+    + Media Content
+    + User-Generated Data
+    + Machine-Learning Models
+  - Major Software Licenses and License Categories
+
+* Community And Ecosystem
+  - Project Archetypes
+  - Community Diversity
+  - Community Health
+  - Commercial Participants
+  - Non-Commercial Participants
+  - Sustainability, Longevity, and Lifecycle
+
+* Policy
+  - Copyright
+  - Trademark
+  - Patent
+  - Security
+  - Data Privacy
+    + GDPR
+  - Aligning FOSS Policy With National/Legal Policy
+
+* Organizational Readiness
+  - Readiness Models
+    + [TODO Group model](https://github.com/todogroup/ospo101/blob/main/module2/README.md#climbing-the-open-source-ladder)
+    + [OTS take on the McAffer Engagemet Model](https://blog.opentechstrategies.com/2020/09/open-source-readiness-models/)
+    + [GitHub's Model](https://github.com/github/maturity-model)
+    + [Qalipso's OMM](https://en.wikipedia.org/wiki/OpenSource_Maturity_Model)
+    + Open Source Skills Stack
+      - Economic / Business
+      - Political / Strategic
+      - Social
+      - Licensing
+      - Technical
+  - Climbing The Readiness Curve
+
+* Procurement
+  - Modular Contracting
+  - IP Contract Terms
+  - Open Source Quality Assurance
+  - Staffing
+  - Open Source Solicitation
+
+* Adoptability Assessment
```

Comment:
```suggestion
* Adoptability Assessment
```

Landscape analysis/mapping of technical capacity and gaps

Reply:
@imahgoub Is the "Navigating Existing DPGs" item about searching and findability -- i.e., given a certain need, this item is about how to locate a DPG or DPGs that might meet that need?

Reply:
(I went with the assumption above, and put that new item under Procurement rather than Adoptability Assessment.)